ardcvocabs/src/main/java/au/org/aodn/ardcvocabs/exception/ExtractingPathVersionsException.java

```diff
@@ -0,0 +1,9 @@
+package au.org.aodn.ardcvocabs.exception;
```

Comment:
Can the branch start with the ticket id so {ticketid}-fail-safe-vocabs-refreshing